app/views/settings/confirmation_email/index.html.erb

```diff
@@ -1,10 +1,14 @@
+<% content_for :back_link do %>
+  <%= render MojForms::BackLinkComponent.new(href: settings_submission_index_path) do %>
+    <%=t('actions.back')%><span class="govuk-visually-hidden">to <%=t('settings.submission.heading')%></span>
```

Comment:
Not sure if it matters for screen readers but there is no space between 'Back' and 'to'?

Reply:
👀 good spot! But spaces actually get added in `::before` and `::after` pseudo elements with the `.govuk-visually-hidden` class.